docs/modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,6 @@
 .Docs
 * xref:index.adoc[Overview]
-* xref:setup.adoc[Setting up Your project]
+* xref:setup.adoc[Setting Up Your Project]
```

Comment:
@nventuro I wanted to use title case for this, which mandates lower case "up". https://capitalizemytitle.com/